src/cargo/core/compiler/fingerprint/dep_info.rs

```diff
@@ -667,4 +667,30 @@ mod encoded_dep_info {
     fn round_trip_with_checksums() {
         gen_test(true);
     }
+
+    // The dep-info file in hex view:
+    //
+    // ┌────────┬─────────────────────────┬─────────────────────────┬────────┬────────┐
+    // │00000000│ 01 00 00 00 00 0a 00 00 ┊ 00 73 72 63 2f 6c 69 62 │•⋄⋄⋄⋄_⋄⋄┊⋄src/lib│
+    // │00000010│ 2e 72 73 08 00 00 00 15 ┊ 00 00 00 43 41 52 47 4f │.rs•⋄⋄⋄•┊⋄⋄⋄CARGO│
+    // │00000020│ 5f 43 4f 4e 54 41 49 4e ┊ 45 52 5f 54 45 53 54 53 │_CONTAIN┊ER_TESTS│
+    // │00000030│ 00 0a 00 00 00 43 41 52 ┊ 47 4f 5f 48 4f 4d 45 01 │⋄_⋄⋄⋄CAR┊GO_HOME•│
+    // │00000040│ 11 00 00 00 2f 68 6f 6d ┊ 65 2f 77 68 6c 6f 2f 2e │•⋄⋄⋄/hom┊e/whlo/.│
+    // │00000050│ 63 61 72 67 6f 1a 00 00 ┊ 00 43 41 52 47 4f 5f 50 │cargo•⋄⋄┊⋄CARGO_P│
+    // │00000060│ 55 42 4c 49 43 5f 4e 45 ┊ 54 57 4f 52 4b 5f 54 45 │UBLIC_NE┊TWORK_TE│
+    // │00000070│ 53 54 53 00 19 00 00 00 ┊ 43 41 52 47 4f 5f 52 55 │STS⋄•⋄⋄⋄┊CARGO_RU│
+    // │00000080│ 4e 5f 42 55 49 4c 44 5f ┊ 53 54 44 5f 54 45 53 54 │N_BUILD_┊STD_TEST│
+    // │00000090│ 53 00 1a 00 00 00 43 41 ┊ 52 47 4f 5f 54 45 53 54 │S⋄•⋄⋄⋄CA┊RGO_TEST│
+    // │000000a0│ 5f 44 49 53 41 42 4c 45 ┊ 5f 4e 49 47 48 54 4c 59 │_DISABLE┊_NIGHTLY│
+    // │000000b0│ 00 02 00 00 00 43 49 00 ┊ 1b 00 00 00 52 55 53 54 │⋄•⋄⋄⋄CI⋄┊•⋄⋄⋄RUST│
+    // │000000c0│ 55 50 5f 57 49 4e 44 4f ┊ 57 53 5f 50 41 54 48 5f │UP_WINDO┊WS_PATH_│
+    // │000000d0│ 41 44 44 5f 42 49 4e 00 ┊ 08 00 00 00 54 46 5f 42 │ADD_BIN⋄┊•⋄⋄⋄TF_B│
+    // │000000e0│ 55 49 4c 44 00          ┊                         │UILD⋄   ┊        │
+    // └────────┴─────────────────────────┴─────────────────────────┴────────┴────────┘
+    #[test]
+    #[cfg_attr(not(target_os = "macos"), ignore = "OOM")]
+    fn parse_v0_fingerprint_dep_info() {
+        let data = include_bytes!("dep-lib-cargo_test_macro");
+        assert!(EncodedDepInfo::parse(&data[..]).is_none());
+    }
 }
```

Comment:
If this test isn't updated in the following commit, what is it we are expecting out of this test?